src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/ResidentialAttribute.kt

```diff
@@ -0,0 +1,108 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.locations
+
+/**
+ * Copied from hmpps-locations-inside-prison-api, used to lookup the ResidentialAttributeType from the ResidentialAttributeValue.
+ */
+
+enum class ResidentialAttributeType(
+  val description: String,
+) {
+  USED_FOR("Used for"),
+  LOCATION_ATTRIBUTE("Location attribute"),
+  SANITATION_FITTINGS("Sanitation and fittings"),
+  NON_ASSOCIATIONS("Non-associations"),
+  SECURITY("Supervision and Security"),
+}
+
+enum class ResidentialAttributeValue(
+  val type: ResidentialAttributeType,
+  val description: String,
+) {
+  ANTI_BARRICADE_DOOR(ResidentialAttributeType.SANITATION_FITTINGS, "Anti Barricade Door"),
+  AUDITABLE_CELL_BELL(ResidentialAttributeType.SANITATION_FITTINGS, "Auditable Cell Bell"),
+  FIXED_BED(ResidentialAttributeType.SANITATION_FITTINGS, "Fixed Bed"),
+  METAL_DOOR(ResidentialAttributeType.SANITATION_FITTINGS, "Metal Door"),
```

Comment:
This is really hard to read - is it possible to remove all the `ResidentialAttributeType` references to help?